app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -605,13 +605,12 @@ def send_failed_fraud_email(enrollment:, visited_location_name:)
     end
   end
 
-  def send_please_call_email(enrollment:, visited_location_name:)
+  def send_please_call_email(enrollment:)
     enrollment.user.confirmed_email_addresses.each do |email_address|
       # rubocop:disable IdentityIdp/MailLaterLinter
-      UserMailer.with(user: enrollment.user, email_address: email_address).in_person_please_call(
-        enrollment: enrollment,
-        visited_location_name: visited_location_name,
-      ).deliver_later(**notification_delivery_params(enrollment))
+      UserMailer.with(user: enrollment.user, email_address: email_address).
+        idv_please_call.
+        deliver_later(**notification_delivery_params(enrollment))
```

Comment:
There's a minor 50/50 state risk here--if the 50/50 state persists longer than the `deliver_later` delay (1 hour), it's possible that an old server could pick up the job and call `idv_please_call`, which won't exist. However:

* I'm only 99% sure that's how `deliver_later` works here (i.e. that it re-calls the mailer method later on)
* A 50/50 state lasting that long _shouldn't_ happen. 

I can remove this and do it in a separate deploy if folks want.

Reply:
Just confirming that `deliver_later` does work this way: https://github.com/rails/rails/blob/17a5ae138ad032ecc372be5e509ef0d15711b22f/actionmailer/lib/action_mailer/message_delivery.rb#L151-L152

Reply:
I'm trying to wrap my head around this one. Is there any danger if we discover and issue and need to revert and go from new back to old?

Reply:
That is a good callout. I hadn't considered reverting. I think this ends up looking a lot like adding a new job: `deliver_later` schedules a job that invokes a method on the mailer. If the mailer the job has when it runs does not have the method defined, it will fail.

So I think this will be two PRs / deploys:
1. Add new method with IPP variant as an alias
2. Start calling it for remote unsupervised + update IPP to call it as well, remove alias